src/python_bring_api/bring.py

```diff
@@ -73,8 +73,16 @@ def login(self) -> BringAuthResponse:
             r = requests.post(f'{self.url}bringauth', data=data)
             r.raise_for_status()
         except RequestException as e:
-            _LOGGER.error(f'Exception: Cannot login:\n{traceback.format_exc()}')
-            raise BringRequestException(f'Authentication failed due to request exception.') from e
+            if e.response.status_code == 401: 
+                errmsg = e.response.json()
+                _LOGGER.error(f'Exception: Cannot login: {errmsg['message']}') 

```

Comment:
Why we use once `errmsg['message']` and once `e.response.text`?

Reply:
good question, the API does not send the error message as JSON in this case, it simply returns plain text 🤷‍♂️

Reply:
i mean, when the API returns a 400 error, it returns plain text instead of JSON